backend/src/main/scala/bloop/tracing/BraveTracer.scala

```diff
@@ -4,10 +4,9 @@ import brave.{Span, Tracer}
 import brave.propagation.TraceContext
 import monix.eval.Task
 import monix.execution.misc.NonFatal
-
-import scala.util.Failure
-import scala.util.Success
+import scala.util.{Failure, Properties, Success}
```

Comment:
nit: please configure IntelliJ to not group imports by prefix

Reply:
done